openpype/hosts/houdini/api/lib.py

```diff
@@ -1054,3 +1054,71 @@ def add_self_publish_button(node):
     template = node.parmTemplateGroup()
     template.insertBefore((0,), button_parm)
     node.setParmTemplateGroup(template)
+
+
+def get_scene_viewer():
+    """
+    return an instance of a visible viewport.
+    There may be many, some could be closed, any visible are current
+    """
+    panes = hou.ui.paneTabs()
+    panes = [x for x in panes if x.type() == hou.paneTabType.SceneViewer]
+    panes = sorted(panes, key=lambda x: x.isCurrentTab())
+    if panes:
+        return panes[-1]
+
+    return None
+
+
+def sceneview_snapshot(
+        sceneview,
+        filepath="$HIP/thumbnails/$HIPNAME.$F4.jpg",
+        fstart=None,
```

Comment:
This is me being annoying. Could full name variables be used, `fstart` > `frame_start` & `fend` > `frame_end`?

Reply:
Fully agree myself, prefer the clarity myself too.